pkg/network/network.go

```diff
@@ -14,12 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package config
+package network
 
 import (
 	"net"
 	"strings"
 
+	"github.com/knative/serving/pkg/reconciler/v1alpha1/clusteringress"
```

Comment:
Can we move the constant into this package and reverse the dependency instead?  I don't like the ideal of using reconcilers as libraries very much.